client/src/components/hero_order/heroOrderCtrl.js

```diff
@@ -16,18 +18,19 @@
     vm.transactionId = 1;
 
 
-    var checkOrder = $interval(isOrderComplete, 2000, 0, false);
+    var checkOrder = $interval(isOrderComplete, 5000, 0, false);
 
 
     function isOrderComplete() {
       ajaxFactory.isOrderComplete(vm.transactionId)
         .then(function(response) {
+          console.log('Server said', response.data);
           if(response.data === true){

```

Comment:
need space before and after parentheses for conditionals
should be `if (response.data === true) {`
This should also be written as `if (response.data) {`